docs/doc/14-sql-commands/10-dml/dml-copy-into-table.md

```diff
@@ -208,6 +208,19 @@ copyOptions ::=
 When importing large volumes of data, such as logs, it is recommended to set both `PURGE` and `FORCE` to True. This ensures efficient data import without the need for interaction with the Meta server (updating the copied-files set). However, it is important to be aware that this may lead to duplicate data imports.
 :::
 
+### Output
+
+The command returns the following columns:
```

Comment:
```suggestion
COPY INTO provides a summary of the data loading results with these columns:
```

Reply:
COPY INTO （table）and COPY INTO （stage ）have diff schemas